lightkurve/search.py

```diff
@@ -711,12 +711,16 @@ def _query_mast(target, radius=None, project=('Kepler', 'K2', 'TESS')):
         # `target_name` under which MAST will know the object to prevent
         # source confusion (see GitHub issue #148).
         target = int(target)
```

Comment:
Problem: if `target` is a string with epic/kic prefix, the line above will raise a `ValueError` and the code below will be skipped until the end of the `except` block we're in.  This is obscure but intentional.